### PR TITLE
#259 added test for (MTB-PL1-BC-CPI-2)

### PR DESCRIPTION
**Pull request by phoenixking25**

### CODE REVIEW

File: benchmarks/e2e/tests/block_multitenant_resources/block_multitenant_resources.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
+package test
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/onsi/ginkgo"
+	configutil "github.com/realshuting/multi-tenancy/benchmarks/e2e/config"
```
Comment (JimBugwadia) [inline benchmarks/e2e/tests/block_multitenant_resources/block_multitenant_resources.go]: @phoenixking25 - this should refer to https://github.com/kubernetes-sigs/multi-tenancy/blob/master/benchmarks/e2e/config/config.go

File: benchmarks/e2e/tests/e2e.go [hunk 2]
```diff
@@ -11,6 +11,7 @@ import (
 	// test sources
 	_ "github.com/realshuting/multi-tenancy/benchmarks/e2e/tests/block_cluster_resources"
 	_ "github.com/realshuting/multi-tenancy/benchmarks/e2e/tests/configure_ns_quotas"
+	_ "github.com/realshuting/multi-tenancy/benchmarks/e2e/tests/block_multitenant_resources"
 )
```
Comment (JimBugwadia) [inline benchmarks/e2e/tests/e2e.go]: Can we make these paths relative? Currently these seem to be referring to a forked repo...

File: benchmarks/e2e/tests/block_multitenant_resources/block_multitenant_resources.go (hunk 1 above)
Comment (phoenixking25) [reply benchmarks/e2e/tests/block_multitenant_resources/block_multitenant_resources.go]: https://github.com/kubernetes-sigs/multi-tenancy/pull/538
@JimBugwadia sure, to run the tests i had to put the path like this before the given merge, i was just waiting for that merge so my pr dont create conflict

File: benchmarks/e2e/tests/e2e.go (hunk 2 above)
Comment (phoenixking25) [reply benchmarks/e2e/tests/e2e.go]: I will new commit in each pr correcting this ASAP

Comment (phoenixking25) [reply benchmarks/e2e/tests/e2e.go]: updated

**Comment by phoenixking25** [comment]
> @phoenixking25 - please rebase / fix the the merge conflict for benchmarks/e2e/tests/e2e.go.

done

**Pull request merged**